src/ui/dockpanel.ts

```diff
@@ -191,7 +191,7 @@ class DockPanel extends Widget {
    *
    * #### Notes
    * The current widget is the widget among the added widgets which
-   * has the *descendant node* which has most recently been focused.
+   * has the *descendant node* which is currently focused.
```

Comment:
What about a dockpanel that doesn't have current focus at all? It should still be able to provide the currently showed widget, no?